_field-types/supported-field-types/star-tree.md

```diff
@@ -0,0 +1,148 @@
+---
+layout: default
+title: Star Tree
+nav_order: 61
+has_children: false
+parent: Supported field types
+redirect_from:
+  - /opensearch/supported-field-types/star-tree/
+  - /field-types/star-tree/
+---
+# Star tree field type
+
+This is an experimental feature and is not recommended for use in a production environment. For updates on the progress the feature or if you want to leave feedback, join the discussion on the [OpenSearch forum](https://forum.opensearch.org/).    
+{: .warning}
+
+Star Tree Index is a multi-field index that improves the performance of aggregations.
+Once you configure star-tree index as part of index mapping by specifying the dimensions and metrics, star-tree index gets created and maintained in real-time within segments as data is ingested.
```

Comment:
dimensions and metrics are used 2 times before there is an explanation for them. 

Maybe we can just mention briefly define them before or use a different statement like 
```
A star-tree index is maintained in real-time within segments during ingestion after it's created.
```